gini/gini-vision-lib-ios
# Added a note in README.md about adding Gini's podspec as a source

# Introduction

Our installation guide in `README.md` wasn't mentioning  that we use a custom podspec repository. 

# How to test

Open the README, scroll to the Cocoapods section and verify that the text acknowledges that we host our podspec ourselves.

# Merge information

Automatic
